docs/config.toml

```diff
@@ -0,0 +1,93 @@
+baseURL= ""
```

Comment:
I think we need to remove doc at all

Reply:
https://github.com/percona/percona-postgresql-operator/pull/309/commits/120b2b9689a7b719975c93904ba71db5a2ffe42e